doc/devel/doc.rst

```diff
@@ -0,0 +1,8 @@
+Generate documentation
+######################
```

Comment:
The is a first-level heading, so it needs to use `#` above and below the text following our style guide.

For more information, see the [RestructuredText Style Guide on headings](https://developer.lsst.io/restructuredtext/style.html#sections).

Reply:
:+1: